pkg/trace/README.md

```diff
@@ -8,6 +8,10 @@ change the config.toml to store traces in the .celestia-app/data/traces
 directory.
 
 ```toml
+# The tracer url token to use for remote event collection.
+# If empty, remote event pulling is disabled.
+trace_pull_address = ":26661"
```

Comment:
TBH I don't understand what "url token" means here. The value looks like a port. Can the comment be made more clear?

If the config is supposed to be a value, why is the address portion omitted and only the port there?

Reply:
that was just old docs, but good catch fixed!